eva/executor/groupby_executor.py

```diff
@@ -0,0 +1,63 @@
+# coding=utf-8
+# Copyright 2018-2022 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import re
+from typing import Iterator
+
+import pandas as pd
+
+from eva.executor.abstract_executor import AbstractExecutor
+from eva.models.storage.batch import Batch
+from eva.planner.groupby_plan import GroupByPlan
+
+
+class GroupByExecutor(AbstractExecutor):
+    """
+    Samples uniformly from the rows.
+
+    Arguments:
```

Comment:
Move to init docstring

Reply:
Followed other executors for consistency.